phpunit.xml.dist

```diff
@@ -41,4 +41,9 @@
 			<group>ms-required</group>
 		</exclude>
 	</groups>
+	<coverage>
+    <include>
+        <directory suffix=".php">plugins</directory>
+    </include>
+</coverage>
```

Comment:
Seems like indentation is off here?

Also @sarthak-19, have you tried generating HTML coverage reports with this config? My speculation is, this config will generate coverage reports in the respect of whole codebase in the plugins directory instead in the respect of a single plugin.

cc @westonruter

Reply:
> Seems like indentation is off here?
I'll fix this.

> have you tried generating HTML coverage reports with this config? My speculation is, this config will generate coverage reports in the respect of whole codebase in the plugins directory instead in the respect of a single plugin.

No I have just tried html generation for single plugin, I'll try once what you are suggesting and will update here whatever seems a viable approach.